conf/yoe.conf

```diff
@@ -0,0 +1,57 @@
+# CONF_VERSION is increased each time build/conf/ changes incompatibly
+CONF_VERSION = "1"
+
+# Modify the following if you need to change the types of images generated
+# most of the time, this is set by the machine conf or in a custom image
+# recipe.
+IMAGE_FSTYPES_append = " tar.gz wic.xz"
```

Comment:
Do we need tar.gz ? and May be just override it, drop appending to defaults.

Reply:
tar.gz is useful for peeking what is in the rootfs -- there may be better ways to do this.

Reply:
ah, I usually go into build tree for image recipe and look inside rootfs/ folder. but that does not fix the issue when you only have images around.